telegram/ext/_jobqueue.py

```diff
@@ -460,6 +460,11 @@ async def callback(context: CallbackContext)
 
                 .. versionchanged:: 20.0
                     Changed day of the week mapping of 0-6 from monday-sunday to sunday-saturday.
+
+                .. versionchanged:: 20.1
+                    The warning about the change of day of week mapping is now only shown if this
+                    parameter was passed.
+
```

Comment:
Is that worth a versionchanged? The warning isn't documented behavior …

Reply:
don't know.. guess I'll remove